src/repositories/timesheet/models.rs

```diff
@@ -0,0 +1,67 @@
+use chrono::{NaiveDate, NaiveDateTime};
+use sqlx::prelude::FromRow;
+use uuid::Uuid;
+
+#[derive(Debug, FromRow)]
+pub struct TimesheetDb {
```

Comment:
Heads up, there is a new field that was added to this.
See timesheet here:
https://github.com/czFIRE/Rust_Organisation_App/pull/41/files#diff-ea6451a3a05f49cb99cfb8b0b736e1d9785e700f66be23e38109b74b96d87451